mne/viz/topo.py

```diff
@@ -632,9 +633,13 @@ def _rm_cursor(event):
             ax.set_ylabel(y_label)
 
     if vline:
-        plt.axvline(vline, color=hvline_color, linewidth=1.0, linestyle="--")
+        vline = [vline] if isinstance(vline, Real) else vline
```

Comment:
For stuff like this we should probably continue to use `mne.utils._is_numeric`

And if you're interested in fixing other stuff in this PR, the `if vline` has me worried that `vline=0.` won't be handled properly...

Reply:
Done, good catch for `vline=0`.
`_is_numeric` is just a 1-line wrapper around `return isinstance(n, numbers.Number)`, preent in 24 lines across 10 files. How about removing this extra function for the `isinstance(...)` directly? It seems like an unnecessary wrapper.

Reply:
Over the years we have changed which check we use like `numbers.Number`, now you're proposing `Real`, later we might want it to be something else. Having our own `_is_numeric` wrapper makes it so we can change this in one place rather than dozens